### PR TITLE
JIT ARM64-SVE: Add BV_2B

### PR DESCRIPTION
**Pull request by amanasifkhalid**
Forgot about this one... Part of #94549. Adds the IF_SVE_BV_2B encoding format.

cstool output:
```
mov   z0.h, p1/m, #0
mov   z2.s, p3/m, #0
mov   z4.d, p5/m, #0
```

JitDisasm output:
```
mov     z0.h, p1/m, #0
mov     z2.s, p3/m, #0
mov     z4.d, p5/m, #0
```

cc @dotnet/arm64-contrib.

### CODE REVIEW

File: src/coreclr/jit/emitarm64.cpp [hunk 1]
```diff
@@ -1193,6 +1193,12 @@ void emitter::emitInsSanityCheck(instrDesc* id)
             assert(isValidSimm8(imm));                                        // iiiiiiii
             break;
 
+        case IF_SVE_BV_2B: // ........xx..gggg ...........ddddd -- SVE copy integer immediate (predicated)
+            assert(insOptsScalableStandard(id->idInsOpt())); // xx
```
Comment (a74nh) [inline src/coreclr/jit/emitarm64.cpp:1197]: INS_OPTS_SCALABLE_B shouldn't be valid here.

File: src/coreclr/jit/emitarm64.cpp [hunk 2]
```diff
@@ -9311,6 +9317,17 @@ void emitter::emitIns_R_R(instruction     ins,
...
 
+        case INS_sve_fmov:
+            assert(insOptsScalableAtLeastHalf(opt));
+            assert(isVectorRegister(reg1));                        // ddddd
+            assert(isPredicateRegister(reg2));                     // gggg
+            assert(isValidVectorElemsize(optGetSveElemsize(opt))); // xx
+            fmt = IF_SVE_BV_2B;
+
+            // MOV is an alias for FMOV, and is always the preferred disassembly.
+            ins = INS_sve_mov;
```
Comment (a74nh) [inline src/coreclr/jit/emitarm64.cpp:9328]: This is odd. The docs suggest it should show fmov: https://docsmirror.github.io/A64/2023-09/fmov_cpy_z_p_i.html

Comment (a74nh) [reply src/coreclr/jit/emitarm64.cpp:9328]: Unhelpfully, that's only on the CPY page in the docs, not FMOV. A comment would be great.

Comment (amanasifkhalid) [reply src/coreclr/jit/emitarm64.cpp:9328]: Sure thing; fixed.

**Review by kunalspathak — APPROVED** [review]
LGTM

**Pull request merged**